src/result.rs

```diff
@@ -171,47 +181,47 @@ impl CqlValueWrapper {
     // Is this correct?
     match self.internal.as_counter() {
       Some(r) => Ok(r.0.into()),
-      None => Err(Error::new(Status::GenericFailure, "Error")),
+      None => Err(Self::generic_error("counter")),
     }
```

Comment:
Regarding the comment: what's the doubt here?

Reply:
I wasn't able to find any usage of Counter type in the js driver. Probably will need to address, but maybe not in this PR

Reply:
So the doubt arises because there is no way to check the logic with the JS driver, because it's not triggered anywhere? Did I get it right?

Reply:
Sort of. It's possibile this type is not used by the DS library so we will be adding it instead of updating as f.ex. duration type